src/string/multiply.rs

```diff
@@ -0,0 +1,54 @@
+/*
+Given two non-negative integers num1 and num2 expressed as strings, returns the product of num1 and num2, which is also expressed as a string.
+Note: You cannot use any of the built-in BigInteger libraries or directly convert the input to an integer.
```

Comment:
This is a _problem statement_. Please change it to a proper _documentation_ (i.e. an explanation what does your code do)